test/install_command.bats

```diff
@@ -272,15 +272,15 @@ EOM
 @test "install_command keeps the download directory when always_keep_download setting is true" {
   echo 'always_keep_download = yes' >"$HOME/.asdfrc"
   run asdf install dummy 1.1.0
-  echo $output
+  echo "$output"
   [ "$status" -eq 0 ]
   [ -d "$ASDF_DIR/downloads/dummy/1.1.0" ]
   [ "$(cat "$ASDF_DIR/installs/dummy/1.1.0/version")" = "1.1.0" ]
 }
 
 @test "install_command fails when download script exits with non-zero code" {
   run asdf install dummy-broken 1.0.0
-  echo $output
+  echo "$output"
```

Comment:
These could probably be deleted.

Reply:
@hyperupcall can you PR the removal of all printing of `output`s as one of your many fix PRs? thanks!

Reply:
def, i'll make a separate PR for that

Reply:
Proposed in #1455